raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -24,7 +24,6 @@
 @pytest.mark.parametrize("deposit", [10])
 @pytest.mark.parametrize("channels_per_node", [CHAIN])
 @pytest.mark.parametrize("number_of_nodes", [2])
-@pytest.mark.skip("Issue: #4312")
```

Comment:
Lol this one seems to have even been [fixed](https://github.com/raiden-network/raiden/issues/4312#issuecomment-523335989) **AND** closed back in August but nobody unskipped the test.